src/seismometer/seismogram.py

```diff
@@ -179,27 +190,27 @@ def target_event(self, event: str):
 
     @property
     def target_cols(self) -> list:
-        return self.config.targets
+        return list(self.config.targets)
```

Comment:
Since you are doing a conversion from a dictionary to a list, are these the column names, or event descriptors (the display names)?

This might be worth calling out in a doc string for this property

Reply:
Good call. Made it clear these are the latter, the display_name from the event